WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/Container/AddProductWithAIContainerViewModel.swift

```diff
@@ -37,6 +39,10 @@ final class AddProductWithAIContainerViewModel: ObservableObject {
     private(set) var productDescription: String?
     private var isFirstAttemptGeneratingDetails: Bool
 
+    private(set) lazy var startingInfoViewModel: ProductCreationAIStartingInfoViewModel = {
+        .init(siteID: siteID)
+    }()
```

Comment:
Nit: we should migrate away from `.init(...)` as a direct init would perform better.

 ```suggestion
    private(set) lazy var startingInfoViewModel = ProductCreationAIStartingInfoViewModel(siteID: siteID)
```

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/13171/commits/1874e8b28f6330c68124709fba7618937f7ca206